environment/src/main/java/com/sequenceiq/environment/environment/flow/stop/event/EnvStopEvent.java

```diff
@@ -0,0 +1,64 @@
+package com.sequenceiq.environment.environment.flow.stop.event;
+
+import com.sequenceiq.flow.reactor.api.event.BaseNamedFlowEvent;
+
+import reactor.rx.Promise;
+
+public class EnvStopEvent extends BaseNamedFlowEvent {
+
+    public EnvStopEvent(String selector, Long resourceId, String resourceName, String resourceCrn) {
```

Comment:
create a constructor that takes a builder

Reply:
this is the convention in this service